.github/workflows/ci.yml

```diff
@@ -27,15 +28,25 @@ jobs:
       run:
         working-directory: ${{ github.workspace }}/src/github.com/getsentry/sentry-go
     steps:
+      - uses: actions/setup-go@v2
+        if: matrix.go-version != 'tip'
+        with:
+          go-version: ${{ matrix.go-version}}
+      - name: Run setup-go for tip
+        if: matrix.go-version == 'tip'
+        working-directory: ${{ github.workspace }}
```

Comment:
I need to define `working-directory` since `${{ github.workspace }}/src/github.com/getsentry/sentry-go` is only created once `actions/checkout` executes.

I had to move the checkout of the `sentry-go` repository *after* building Go because I would get a [build error](https://github.com/armenzg/sentry-go/runs/1697470150):
```
Run git clone https://go.googlesource.com/go gotip
  git clone https://go.googlesource.com/go gotip
  cd gotip/src
  export GOROOT=$GITHUB_WORKSPACE/gotip
  ./make.bash
  echo "GOROOT=$GOROOT" >> $GITHUB_ENV
  echo "PATH=$GOROOT/bin:$PATH" >> $GITHUB_ENV
  shell: /bin/bash -e {0}
  env:
    GO111MODULE: off
    GOPATH: /home/runner/work/sentry-go/sentry-go
Cloning into 'gotip'...
Building Go cmd/dist using /opt/hostedtoolcache/go/1.14.13/x64. (go1.14.13 linux/amd64)
go tool dist: found go.mod file in /home/runner/work/sentry-go/sentry-go/src/github.com/getsentry/sentry-go: $GOROOT must not be inside a module
Error: Process completed with exit code 1.
```
>